ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -928,256 +897,162 @@ checks:
       - More detailed documentation on OpenLDAP is available at https://www.openldap.org
     condition: none
     rules:
-      - 'c:systemctl is-enabled slapd ->  r:enabled'
+      - 'p:slapd'
 
 
 
-# 2.1.7 Ensure nfs-utils is not installed or the nfs-server service is masked (Automated)
-  - id: 20551 
+# 2.1.17 Ensure nfs-utils is not installed or the nfs-server service is masked (Automated)
+  - id: 20551
```

Comment:
This check doesn't match with the 2.1.17 of the benchmark (`Ensure nfs-utils is not installed or the nfs-server service is masked`). Neither do the rules.